src/app/modules/submission/email-submission/email-submission.controller.ts

```diff
@@ -133,34 +115,38 @@ const submitEmailModeForm: ControllerHandler<
       .andThen((form) => {
         // Check the captcha
         if (form.hasCaptcha) {
-          if (enableTurnstileFeatureFlag) {
-            return TurnstileService.verifyTurnstileResponse(
-              req.query.captchaResponse,
-              getRequestIp(req),
-            )
-              .map(() => form)
-              .mapErr((error) => {
-                logger.error({
-                  message: 'Error while verifying turnstile captcha',
-                  meta: logMeta,
-                  error,
+          switch (req.query.captchaType) {
+            case CaptchaTypes.Turnstile: {
+              return TurnstileService.verifyTurnstileResponse(
+                req.query.captchaResponse,
+                getRequestIp(req),
+              )
+                .map(() => form)
+                .mapErr((error) => {
+                  logger.error({
+                    message: 'Error while verifying turnstile captcha',
+                    meta: logMeta,
+                    error,
+                  })
+                  return error
                 })
-                return error
-              })
-          } else {
-            return CaptchaService.verifyCaptchaResponse(
-              req.query.captchaResponse,
-              getRequestIp(req),
-            )
-              .map(() => form)
-              .mapErr((error) => {
-                logger.error({
-                  message: 'Error while verifying captcha',
-                  meta: logMeta,
-                  error,
+            }
+            // defaults to recaptcha
```

Comment:
/nit wondering if we should be explicit and declare `CaptchaTypes.Captcha` as fall-through to the default.

Reply:
Considered defining CaptchaTypes.Captcha explicitly as a case but I wanted to fallback to recaptcha regardless of types